ggshield/output/text/iac_text_output_handler.py

```diff
@@ -51,19 +52,31 @@ def process_iac_file_result(
 
         result_buf.write(file_info(file_result.filename, len(file_result.incidents)))
 
-        file = File.from_bytes(file_path.read_bytes(), str(file_path))
-        lines: List[Line] = get_lines_from_content(file.document, Filemode.FILE, False)
+        try:
```

Comment:
I'm not sure this is really useful, something causing this kind of error (where a file was read and sent to the API, but could not be read afterwards) would probably have much worse consequences than having ggshield crash anyway.

Reply:
It can happen if we have a change in the API in the file format for example and not only if the file does not exist. I would prefer not to fail unexpectedly when there are results from the API, especially since errors are not fully wrapped in ggshield.

Reply:
It can happen if we have a change in the API in the file format for example and not only if the file does not exist. I would prefer not to fail unexpectedly when there are results from the API, especially since errors are not fully wrapped in ggshield.